docs/versioning.rst

```diff
@@ -0,0 +1,96 @@
+**********
+Versioning
+**********
+
+
+Current Status
+==============
+
+When a new major version is released it becomes the current release line, where all new features, bug fixes and security patches land.
+
+
+.. list-table:: Release Lines
+   :widths: 20 30 30 20
+   :header-rows: 1
+
+   * - Line
+     - Supported Runtimes
+     - Minimum Trace Agent
+     - Status
+   * - 0.x
+     - CPython: 2.7, 3.5+
+     - 
+     - MAINTENANCE [2022-02-25]
+   * - 1.x
+     - CPython: 2.7, 3.5+
+     - 7.22
```

Comment:
```suggestion
     - 7.28
```

This version if almost 1 year old, and is min needed to support stats computation